parmed/charmm/parameters.py

```diff
@@ -446,8 +455,17 @@ def read_parameter_file(self, pfile, comments=None):
                     raise CharmmError('Could not parse angles.')
 
                 angle_type = AngleType(k, theteq)
-                self.angle_types[(type1, type2, type3)] = angle_type
-                self.angle_types[(type3, type2, type1)] = angle_type
+                if (type1, type2, type3) in self.angle_types:
+                    # See if the existing angle type list has a different value and replaces it with a warning
+                    if self.angle_types[(type1, type2, type3)] != angle_type:
+                        # Replace. Warn if they are differen
+                        warnings.warn('Replacing angle %r, %r with %r' %
+                                              ([(type1, type2, type3)], self.angle_types[(type1, type2, type3)], angle_type))
```

Comment:
Warn with `ParameterWarning`.  Also, can you try cleaning up the formatting a bit so the lines don't run quite as long?